modular_skyrat/modules/blueshield/code/blueshield.dm

```diff
@@ -65,6 +61,8 @@
 	box = /obj/item/storage/box/survival/security
 	belt = /obj/item/modular_computer/pda/security
 
+	l_hand = /obj/item/storage/toolbox/guncase/skyrat/carwo_large_case/sindano
```

Comment:
Please leave it in the backpack, having it in-hand just risks it being either deleted or overriden by something else much more frequently.

Reply:
The case is too large to fit in bags. Outfit sanity would fail and it makes sense for a full size firearm case to not fit in a bag.

Reply:
Actually I'll just put it in the blueshield locker.